test/expected/parallel.out

```diff
@@ -3,6 +3,10 @@
 CREATE TABLE test (i int, j double precision, ts timestamp);
 --has to be big enough to force at least 2 workers below.
 INSERT INTO test SELECT x, x+0.1, _timescaledb_internal.to_timestamp(x*1000)  FROM generate_series(1,1000000) AS x;
+SET client_min_messages = 'error';
+--avoid warning polluting output
+ANALYZE;
```

Comment:
This analyze just add test stability since autovacuums don't affect result anymore